internal/cmd/providers.go

```diff
@@ -29,13 +28,18 @@ func providersCmd() *cobra.Command {
 }
 
 func createProviderCmd() *cobra.Command {
+	var (
+		name           string
+		hostname       string
+		clusterAddress string
+	)
 	cmd := &cobra.Command{
-		Use:   "create [workspace_provider_name]",
+		Use:   "create --name=[name] --hostname=[hostname] --clusterAddress=[clusterAddress]",
```

Comment:
for consistency we may want to do `name` as a required positional argument, similar to `coder envs create`